packages/utxo-lib/tsconfig.json

```diff
@@ -1,8 +1,6 @@
 {
     "extends": "../../tsconfig.base.json",
-    "compilerOptions": {
-        "outDir": "./libDev"
-    },
+    "compilerOptions": { "outDir": "./libDev" },
```

Comment:
Probably unintentional right? This should be reverted

Reply:
Done